src/Libraries/Microsoft.Extensions.Http.Resilience/Microsoft.Extensions.Http.Resilience.csproj

```diff
@@ -40,70 +40,26 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="buildTransitive\$(MSBuildProjectName).*"
+    <None Include="buildTransitive\$(MSBuildProjectName).targets"
```

Comment:
This doesn't look correct. What about the props?

Reply:
You suggested above (https://github.com/dotnet/extensions/pull/5363#discussion_r1741274520) to move the whole PropertyGroup to the .targets file. I moved the group and removed the .props file. We now have only the .targets file, therefore we copy only it. Right?

Reply:
Ah yes, indeed.
Though, it we add a props file, we may forget to change this. Suggest keeping it as `*`.

Reply:
Did as you suggested. Now we copy all files, i.e. use `.*`.